<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>974</width>
    <height>643</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QHBoxLayout" >
    <item>
     <widget class="GLViews" native="1" name="view" />
    </item>
    <item>
     <widget class="QSlider" name="WindowCenter" >
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="WindowWidth" >
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="WindowReject" >
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>974</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuStudies" >
    <property name="title" >
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpen" />
    <addaction name="actionOpen_DICOMDIR" />
    <addaction name="actionOpen_PACS" />
    <addaction name="action_Export" />
    <addaction name="actionQuit" />
   </widget>
   <widget class="QMenu" name="menuSettings" >
    <property name="title" >
     <string>Settings</string>
    </property>
    <addaction name="actionSettingsPACS" />
   </widget>
   <widget class="QMenu" name="menuVOI" >
    <property name="title" >
     <string>VOI</string>
    </property>
    <addaction name="actionAuto" />
    <addaction name="actionNone" />
    <addaction name="actionPreset" />
   </widget>
   <addaction name="menuStudies" />
   <addaction name="menuSettings" />
   <addaction name="menuVOI" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <widget class="QToolBar" name="toolBar" >
   <property name="windowTitle" >
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen" />
   <addaction name="actionOpen_DICOMDIR" />
   <addaction name="actionOpen_PACS" />
   <addaction name="action_Export" />
  </widget>
  <action name="actionOpen" >
   <property name="icon" >
    <iconset resource="cyDICOM.qrc" >:/icons/icons/fileopen.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Open files</string>
   </property>
   <property name="toolTip" >
    <string>Open studies from files</string>
   </property>
  </action>
  <action name="actionQuit" >
   <property name="icon" >
    <iconset resource="cyDICOM.qrc" >:/icons/icons/exit.png</iconset>
   </property>
   <property name="text" >
    <string>Quit</string>
   </property>
   <property name="toolTip" >
    <string>Quit application</string>
   </property>
  </action>
  <action name="actionOpen_DICOMDIR" >
   <property name="icon" >
    <iconset resource="cyDICOM.qrc" >:/icons/icons/fileimport.png</iconset>
   </property>
   <property name="text" >
    <string>Open &amp;DICOMDIR</string>
   </property>
   <property name="toolTip" >
    <string>Open studies from DICOMDIR</string>
   </property>
  </action>
  <action name="actionOpen_PACS" >
   <property name="icon" >
    <iconset resource="cyDICOM.qrc" >:/icons/icons/system.png</iconset>
   </property>
   <property name="text" >
    <string>Open &amp;PACS</string>
   </property>
   <property name="toolTip" >
    <string>Open studies from PACS</string>
   </property>
  </action>
  <action name="action_Export" >
   <property name="icon" >
    <iconset resource="cyDICOM.qrc" >:/icons/icons/fileexport.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Export</string>
   </property>
   <property name="toolTip" >
    <string>Export series to DICOM</string>
   </property>
  </action>
  <action name="actionSettingsPACS" >
   <property name="text" >
    <string>PACS</string>
   </property>
  </action>
  <action name="actionAuto" >
   <property name="text" >
    <string>Auto</string>
   </property>
  </action>
  <action name="actionNone" >
   <property name="text" >
    <string>None</string>
   </property>
  </action>
  <action name="actionPreset" >
   <property name="text" >
    <string>Preset</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GLViews</class>
   <extends>QWidget</extends>
   <header>glviews.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="cyDICOM.qrc" />
 </resources>
 <connections/>
</ui>
